language/ir-testsuite/tests/misc/ignore_checks_when_exp_file_present.exp

```diff
@@ -0,0 +1,9 @@
+[0] Transaction(0)
+[1] Stage(Compiler)
+[2] Output(CompilerLog("Script(\nImports()\nDependency(\n)\nConstants(\n)\nMain(() (return ();\n\n)))"))
+[3] Output(CompiledScript(CompiledScript(CompiledScriptMut { module_handles: [], struct_handles: [], function_handles: [], function_instantiations: [], signatures: [Signature([])], identifiers: [], address_identifiers: [], constant_pool: [], type_parameters: [], parameters: SignatureIndex(0), code: CodeUnit { locals: SignatureIndex(0), code: [Ret] } })))
+[4] Stage(Verifier)
+[5] Stage(Serializer)
+[6] Stage(Runtime)
+[7] Transaction Status: Keep(EXECUTED)
+[8] Status(Success)
```

Comment:
Sorry, I should have been more explicit in my last review. I don't think we should output anything here but step 7. We can opt into the rest. 

Ideally, the exp file for this test would look like. 

```
Transaction 0 (Starts at line 1): 

Transaction Status: Keep(EXECUTED)
```

Reply:
Normally, I wouldn't care, but the compiler output stuff is just so crazy, that I'm worried it will hurt the ability to quickly approve PRs

Reply:
As another note, thoughts about by default also printing out the VM Error code? (not the status code)

Reply:
I've done both.